.github/workflows/check-release.yml

```diff
@@ -8,7 +8,6 @@ on:
 jobs:
   check-version:
     runs-on: ubuntu-latest
-    if: startsWith(github.head_ref, 'release/') || startsWith(github.ref, 'refs/heads/release/')
```

Comment:
This and all the rest of the python changes are already on main. I checked main and I don't know why it's showing it as new code. Any idea ?